app-backend/http4s-util/src/main/scala/com/rasterfoundry/http4s/xray/XRayTracer.scala

```diff
@@ -0,0 +1,38 @@
+package com.rasterfoundry.http4s
+
+import cats.effect._
+import com.colisweb.tracing.TracingContext._
+import com.amazonaws.services.xray.{AWSXRayAsync, AWSXRayAsyncClientBuilder}
+import com.rasterfoundry.http4s.xray._
+
+object XRayTracer {
+
+  def getTracer: AWSXRayAsync = {
+    AWSXRayAsyncClientBuilder.defaultClient()
+  }
+
+  trait XRayTracingContextBuilder[F[_]] extends TracingContextBuilder[F] {
```

Comment:
Created a separate context builder so that it can have an `apply` method that takes an optional `XrayHttp` object that adds some AWS specific metadata that is helpful for examining http requests.